lib/rouge/formatters/html_line_table.rb

```diff
@@ -29,14 +31,15 @@ def initialize(formatter, opts={})
         @code_class   = opts.fetch :code_class,   'rouge-code'
         @line_class   = opts.fetch :line_class,   'lineno'
         @line_id      = opts.fetch :line_id,      'line-%i'
+        @line_attr    = opts.fetch :line_attr,    'id'
       end
 
       def stream(tokens, &b)
         lineno = @start_line - 1
         buffer = [%(<table class="#@table_class"><tbody>)]
         token_lines(tokens) do |line_tokens|
           lineno += 1
-          buffer << %(<tr id="#{sprintf @line_id, lineno}" class="#@line_class">)
+          buffer << %(<tr #@line_attr="#{sprintf @line_id, lineno}" class="#@line_class">)
```

Comment:
While we won't be accepting this PR, I thought it might be helpful to note that this approach may cause problems depending on the browser. Strictly speaking, HTML attributes should not be repeated in a tag. If you use `"class"` as the value of `@line_attr` you may see unexpected behaviour. More details in [this SO answer](https://stackoverflow.com/questions/26341507/can-an-html-element-have-the-same-attribute-twice/26341866#26341866).